src/org/zaproxy/zap/extension/jxbrowserlinux32/resources/Messages.properties

```diff
@@ -0,0 +1,3 @@
+# This file defines the default (English) variants of all of the internationalised messages
+jxbrowserlinux32.button.launch			= Launch the ZAP JxBrowser proxying through ZAP
```

Comment:
Could these messages be moved to base add-on? (same for other OS/arch specific add-ons.)